foundation/foundation-mda/src/main/resources/templates/deployment/spark-operator/spark-application-base-values.yaml.vm

```diff
@@ -9,34 +9,6 @@ sparkApp:
     #end
     mainApplicationFile: "local:///opt/spark/jobs/pipelines/${mainApplicationFile}"
     deps:
-      #if (!${usePackageDeps})
```

Comment:
I: Because we pulled out the logic for supporting older versions of Spark, `usePackageDeps` can be assumed to be `true`.